include/RED4ext/Memory/Allocators.hpp

```diff
@@ -2715,11 +2719,19 @@ struct GPUM_Buffer_MorphTargetsAllocator : Allocator<GPUM_Buffer_MorphTargets>
 
 struct [[deprecated("Use 'Memory::IAllocator' instead.")]] IMemoryAllocator : Memory::IAllocator
 {
-    struct [[deprecated("Use 'Memory::AllocationResult' instead.")]] Result : Memory::AllocationResult{};
+    struct [[deprecated("Use 'Memory::AllocationResult' instead.")]] Result : Memory::AllocationResult
+    {
+    };
 };
 
-struct [[deprecated("Use 'Memory::EngineAllocator' instead.")]] EngineAllocator : Memory::EngineAllocator{};
-struct [[deprecated("Use 'Memory::RTTIAllocator' instead.")]] RTTIAllocator : Memory::RTTIAllocator{};
+struct [[deprecated("Use 'Memory::EngineAllocator' instead.")]] EngineAllocator : Memory::EngineAllocator
+{
+};
+struct [[deprecated("Use 'Memory::RTTIAllocator' instead.")]] RTTIAllocator : Memory::RTTIAllocator
+{
+};
 struct [[deprecated("Use 'Memory::RTTIFunctionAllocator' instead.")]] RTTIFunctionAllocator
-    : Memory::RTTIFunctionAllocator{};
+    : Memory::RTTIFunctionAllocator
+{
+};
```

Comment:
Since you're modifying this file, I would say to remove these. They have been here for years.

Reply:
Thanks!